test/integration/auth.spec.ts

```diff
@@ -137,7 +142,10 @@ describe('admin.auth', () => {
       });
   });
 
-  it('createUser() creates a new user with enrolled second factors', () => {
+  it('createUser() creates a new user with enrolled second factors', function () {
+    if (authEmulatorHost) {
```

Comment:
Too many conditional skips creeping into this test suite. What are the chances the emulator will support these features in the future so we can get rid of these?

Reply:
(discussed internally about unreleased features)